.travis.yml

```diff
@@ -8,14 +8,18 @@ env:
   - TOXENV=py27-1.6.x
   - TOXENV=py27-1.7.x
   - TOXENV=py27-1.8.x
+  - TOXENV=py27-1.9.x
   - TOXENV=py33-1.5.x
   - TOXENV=py33-1.6.x
   - TOXENV=py33-1.7.x
   - TOXENV=py33-1.8.x
+  - TOXENV=py33-1.9.x
```

Comment:
Django 1.9 does not support Python 3.3 https://docs.djangoproject.com/en/1.9/releases/1.9/#python-compatibility